### DISCUSSION TITLE
How to take decisions in Vocabulary Work-group

### DISCUSSION THREAD

**Discussion by erkist** [cdfoundation/sig-events#40] [General]
Hi all!

In the last Vocabulary work-group meeting, we had a question to the wider Events SIG regarding how we in the work-group should/could make decisions.

What we are looking for is some way to be reasonably efficient while also keeping in line with what the SIG as a whole wants.

**How would the SIG want the work-group to take decisions?**

Some alternatives to consider:
1. The work-group prepares input to the decision and brings this up for a hopefully efficient decision at the next SIG meeting.
2. The work-group takes decisions on its own, but shares the outcome with the SIG on the next meeting to allow the decision to be overturned if needed.

**Comment by m-linner-ericsson** [comment]
If we go for option 1 then the workgroup will need to prepare the material in such depth that you should be able to take a decision at the next SIG meeting but state that most of the discussion should be done beforehand.

The reason for my suggestion is to enable people that have little time to attend only the SIG meeting and get the updates needed. Also, the formality will hopefully give us some motivation for documenting the decision in a good way. 

Does this make any sense?

**Reply by erkist** [reply]
Sounds good to me! Documenting input and decisions is also valuable to those who join the workgroup at a later date and want to "read up" on the progress thus far.

**Reply by m-linner-ericsson** [reply]
Should this be documented anywhere?
